ludwig/backend/ray.py

```diff
@@ -812,17 +812,24 @@ def initialize(self):
         # Disable placement groups on dask
         dask.config.set(annotations={"ray_remote_args": {"placement_group": None}})
 
+    def generate_bundles(self, num_cpu):
```

Comment:
Nit: prefix with underscore since this is not really meant to be used outside this package.